code/modules/shuttle/on_move.dm

```diff
@@ -331,6 +331,18 @@ All ShuttleMove procs go here
 	if(. & MOVE_AREA)
 		. |= MOVE_CONTENTS
 
+/obj/structure/ladder/afterShuttleMove(turf/oldT, list/movement_force, shuttle_dir, shuttle_preferred_direction, move_dir, rotation)
+	. = ..()
+	// be moved, but re-connect at the destination
+	disconnect()
+	LateInitialize()
+
+/obj/structure/ladder/unbreakable/onShuttleMove(turf/newT, turf/oldT, list/movement_force, move_dir, obj/docking_port/stationary/old_dock, obj/docking_port/mobile/moving_dock)
+	// simply don't be moved
+	return FALSE
```

Comment:
does this just silently not move the cargo shuttle? if so, could you add it to blacklisted_cargo_types so there's a warning and people know why it failed?

Reply:
Nothing in the shuttleMove family of procs can stop the shuttle itself